### DISCUSSION TITLE
Option to have reverb (or other future effects) in mono mode

### DISCUSSION THREAD

**Discussion by baconmonsta** [probonopd/MiniDexed#645] [Ideas]
I use Minidexed as 2 synth engines TG1 & TG2" panned hard left and right. The output channels (left & right) go to different destinations in my music setup. If I use reverb on either of the engines, the reverberated sound will start leaking out from the other channel as well, rendering it useless for my use case.
Therefore I'd propose having the option to set reverb into mono-mode (i.e. no stereo widening) to keep the stereo image clean.

**Comment by Banana71** [comment]
I haven't tried it myself but what happens if you set the parameter 
`ReverbDiffusion=0` ?

**Reply by baconmonsta** [reply]
Go ahead and try it! To my understanding diffusion has nothing to do with the stereo image...

**Reply by Banana71** [reply]
I just tried it and it doesn't work. 
The reverb becomes a delay with fixed delay times and the feedback is set using “Size”.
